src/types/store/contact.action.types.ts

```diff
@@ -41,6 +45,15 @@ interface ICreateContactFailedAction {
   type: CREATE_CONTACT_FAILED;
 }
 
+interface IUpdateContactSuccessAction {
+  type: UPDATE_CONTACT_SUCCESS;
+  payload: Array<IContact>;
```

Comment:
should only be 1 contact

Reply:
https://github.com/Sphereon-Opensource/ssi-mobile-wallet/pull/135#discussion_r1191163081

Fixed it as per review comment.